### PR TITLE
Fix "occured" typo

### PR DESCRIPTION
**Pull request by aaronellington**

### CODE REVIEW

File: misc/dist/html/editor.html
Comment (tetrapod00) [inline misc/dist/html/editor.html:709]:
```suggestion
			setStatusNotice('An unknown error occurred.');
```
I'm not sure of the context of these messages, but it's possible that they should be sentence case with a period, if we're already updating them. There is another similar instance.

File: thirdparty/basis_universal/encoder/basisu_comp.cpp
Comment (tetrapod00) [inline thirdparty/basis_universal/encoder/basisu_comp.cpp:3481]:
```suggestion
				// Stop using OpenCL if a failure ever occurs.
```
I don't think this one is right :)

File: misc/dist/html/full-size.html
Comment (tetrapod00) [inline misc/dist/html/full-size.html:143]:
```suggestion
			setStatusNotice('An unknown error occurred.');
```
Possibly this. See other comment for reasoning

**Review by tetrapod00 — COMMENTED** [review]
I can't speak to whether this is a big enough change to be worth a PR, but if it is, it looks like there are several instances of changes to thirdparty files, and also a couple other changes that *might* need to be made.

**Comment by Repiteo** [comment]
I'm shocked that this wasn't caught by codespell; might need to add our own dictionary?

**Comment by tetrapod00** [comment]
It's in the default codespell dictionary (https://github.com/codespell-project/codespell/blob/da60275a48eabf63a01c4dab6eec3c1bc55408e1/codespell_lib/data/dictionary.txt#L40933-L40933):
```
occured->occurred
```
Possibly something wrong with the codespell setup; on the docs repo it was misconfigured for a while and did not catch deeply or shallowly nested files, only files one directory deep. Or maybe it doesn't check in comments or inside snake case variable names? I'm not super familiar with codespell's limits.

**Comment by Repiteo** [comment]
Oh wait, the majority of changed cases were third-party files, and codespell deliberately ignores those. Must not recognize snake-case then, or it needs some special setup; not as big of an issue as I thought in either case

**Review by AThousandShips — DISMISSED** [review]
Third-party files should indeed not be changed

**Comment by aaronellington** [comment]
Thanks for the feedback. I think I've made all of the requested changes. :)

**Comment by tetrapod00** [comment]
Looks reasonable to me now, but still needs approval from the relevant maintainers

**Review by AThousandShips — APPROVED** [review]
This makes sense, but might be better to perform in a bulk change, see:
* https://github.com/godotengine/godot/issues/91521

I'll leave it up to the production team to make the final judgement if this is too small to be appropriate

**Comment by akien-mga** [comment]
As suggested, this kind of changes are best consolidated together with other similar codebase cleanups, so I went ahead and did that in #99799, which includes those changes.

Thanks for your contribution!

**Comment by adamscott** [comment]
> I'm shocked that this wasn't caught by codespell; might need to add our own dictionary?

We should maybe contribute it to codespell.

**Pull request closed without merging**
